deployment/terraform/output.go

```diff
@@ -50,6 +50,14 @@ type Output struct {
 			PrivateDNS string `json:"private_dns"`
 		} `json:"value"`
 	} `json:"metricsServer"`
+	ElasticServer struct {
```

Comment:
Please also show the endpoint info at the end, as part of `(t *Terraform) displayInfo`

Reply:
Done